release-notes/release-notes5105.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: default
+title: TinyMCE 5.10.5
+title_nav: TinyMCE 5.10.5
+description: Release notes for TinyMCE 5.10.5
+keywords: releasenotes bugfixes
+---
+
+## Overview
+
+{{site.productname}} 5.10.5 was released for {{site.enterpriseversion}} and {{site.cloudname}} on Wednesday, June 1<sup>st</sup>, 2022. It includes {{site.productname}} 5.10.4. These release notes provide an overview of the changes for {{site.productname}} 5.10.4, including:
+
+- [General bug fixes](#generalbugfixes)
+- [Upgrading to the latest version of TinyMCE 5](#upgradingtothelatestversionoftinymce5)
+
+{{site.releasenotes_for_stable}}
+
+## General bug fixes
+
+{{site.productname}} 5.10.4 provides fixes for the following bugs:
+
+### base64 encoded images were corrupted and not displayed when added immediately following the string `data:`
+
+When the string `data:` immediately preceded the insertion point, or was only separated from the insertion point by a Return character (ie, was on the line above the insertion point), adding an image in to a {{site.productname}} 5.x document caused the added image data to be parsed incorrectly.
+
+Rather than displaying the encoded image, the mis-parsing caused the image to corrupt and present as a raw string of base64 text within html tags, which also displayed as text.
+
+With this update, the `extractBase64DataUris` function, which was responsible for the mis-parsing, now engages in more comprehensive sanity checking. These further sanity checks prevent the function from mis-parsing a user-entered `data:` string.
```

Comment:
I do wonder if including the function name is a bit overkill? Eg:
```suggestion
With this update, the code which was responsible for the mis-parsing now engages in more comprehensive sanity checking. These further sanity checks prevent the function from mis-parsing a user-entered `data:` string.
```

Reply:
It could well be, but I copied-and-pasted this from the _6.0.3 Release Notes_, so I’m gonna go with _if it’s overkill it’s overkill all the way_ at this point. 😄

Reply:
This wasn't in the 6.0.3 release notes though... this was a 5.10.5 specific change so this is the first time it's being reviewed by others.

Reply:
Thinking out loud (so to speak):

The function name is overkill.

OTOH, it is also specific and concrete.

`code` is, by contrast, generic and non-specific.

I like specific and concrete, but that’s a bias, not an argument.

I can make arguments for the bias — for example, there’s strong evidence that concrete words read as more real and active, in general, compared to generic ones — but it’s still my bias at work.

An edit that, perhaps, brings us to the middle-ground?

-the code which was responsible for the mis-parsing now engages
+the code responsible for the mis-parsing now engages

`code` is still a touch non-specific, but the noun-phrase and near-clause `the code responsible for` is not waffly or indirect at all.